src/vivarium_csu_ltbi/data/baseline_coverage.csv

```diff
@@ -0,0 +1,71 @@
+location,year,treatment_subgroup,value
+South Africa,2019,with_hiv,67.14
```

Comment:
This is the coverage data that will be used --> can you change here with a PR in the future if the data changes ?